modules/apps/layout/layout-seo-web-test/src/testIntegration/java/com/liferay/layout/seo/web/internal/servlet/taglib/test/OpenGraphTopHeadDynamicIncludeTest.java

```diff
@@ -1014,6 +1014,8 @@ public void testIncludeLocalesLayoutTranslatedLanguagesPortletLayout()
 
 		_layout = _layoutLocalService.updateLayout(_layout);
 
+		Set<Locale> locales = new HashSet<>();
```

Comment:
@JanisZhang Did Brian ask to put the declaration at almost the beginning of the method?

Reply:
Hi @dantewang, Brian's original commit message is `SF, As used`.  From my understanding, it means I should make the declared variable next to its usage. After revisiting this commit,  I think this SF is not necessary. I will drop this commit and resend it. Thanks!